packages/gestalt/src/Button.js

```diff
@@ -176,21 +178,16 @@ function Button(props: Props): Element<'button'> {
     </button>
   );
   /* eslint-enable react/button-has-type */
-}
+});
 
-Button.propTypes = {
+// $FlowFixMe Flow(InferError)
```

Comment:
@AlbertCarreras wondering if we can figure out a fix for this. Also can we use specific comments? `// $FlowFixMe [issue]`